Makefile

```diff
@@ -0,0 +1,72 @@
+SERVICE ?= go-aws-config
+REGISTRY ?= pbxx
+COMMIT = $(shell git rev-parse --short HEAD 2>/dev/null)
+LDFLAGS ?=
+DOCKER_BUILD_ARGS ?=
+
+.PHONY: help
+help:
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+.PHONY: analyze
+analyze: fmt lint vet test ## run fmt, lint, vet, and test
+
+.PHONY: build
+build: build-linux ## run go build for linux
+
+.PHONY: build-linux
+build-linux: 
+	GOOS=linux GOARCH=amd64 go build -ldflags "$(LDFLAGS)" -o bin/app src/main/*.go
+
+.PHONY: build-native
+build-native: ## run go build for current OS
+	go build -ldflags "$(LDFLAGS)" -o bin/app src/main/*.go
+
+.PHONY: docker-build
+docker-build: ## build docker image
+	docker build -t $(REGISTRY)/$(SERVICE):$(COMMIT) $(DOCKER_BUILD_ARGS) .
+
+.PHONY: docker-push
```

Comment:
I have a pr to take this out of base service. I just don't think we want this command to be readily available